server/config.go

```diff
@@ -62,3 +64,7 @@ type Profile struct {
 func (prof *Profile) useEmbedDB() bool {
 	return len(prof.PgURL) == 0
 }
+
+func (prof *Profile) isBackendPortReusedForPrometheus() bool {
```

Comment:
My preference is to give only one option and probably reuse to save a port whenever possible.

Reply:
OK. It seems that only /api is allowed to the backend here, so an extra port is really unnecessary.

I'm not sure if the introduction of Prometheus is currently useful, if not maybe the last commit could be removed.

Reply:
I'm fine with introducing "github.com/labstack/echo-contrib/prometheus" since it doesn't add any new dependency.